jax/_src/numpy/ufuncs.py

```diff
@@ -3132,9 +3132,38 @@ def heaviside(x1: ArrayLike, x2: ArrayLike, /) -> Array:
                 _where(lax.gt(x1, zero), _lax_const(x1, 1), x2))
 
 
-@implements(np.hypot, module='numpy')
 @jit
 def hypot(x1: ArrayLike, x2: ArrayLike, /) -> Array:
+  r"""
+  Return element-wise hypotenuse for the given legs of a right angle triangle.
+
+  JAX implementation of :obj:`numpy.hypot`.
+
+  Args:
+    x1: scalar or array. Speicifies one of the legs of right angle triangle.
+      ``complex`` dtype are not supported.
+    x2: scalar or array. Speicifies the other leg of right angle triangle.
+      ``complex`` dtype are not supported. ``x1`` and ``x2`` must either have
+      same shape or be broadcast compatible.
+
+  Returns:
+    An array containing the hypotenuse for the given given legs ``x1`` and ``x2``
+    of a right angle triangle, promoting to inexact dtype.
+
+  Note:
+    ``jnp.hypot`` is equivalent to computing ``jnp.sqrt(x1 ** 2 + x2 **2)``.
```

Comment:
But note that's not entirely true! If you compute it that way, it's numerically unstable.

So you should probably say something like "is a more numerically stable way of computing"

Reply:
Thanks! Updated the note.